test/master-playlist-controller.test.js

```diff
@@ -101,7 +105,10 @@ QUnit.test('creates combined and audio only SegmentLoaders', function() {
               'created alternate audio track segment loader');
 });
 
-QUnit.test('if buffered, will request second segment byte range', function() {
+/* eslint-disable no-warning-comments */
+// TODO add back in once logic is returned in segment-loader
+/* eslint-enable no-warning-comments */
+QUnit.skip('if buffered, will request second segment byte range', function() {
```

Comment:
Yeah, for now. Once logic similar to the timelineUpdated check is added back in, we can return it: https://github.com/videojs/videojs-contrib-hls/pull/619/files#diff-d5c992e639e8f39a5f94c4a3a6582a00R677